receiver/opencensusreceiver/opencensus_test.go

```diff
@@ -612,3 +614,41 @@ func TestOCReceiverMetrics_HandleNextConsumerResponse(t *testing.T) {
 		}
 	}
 }
+
+type errOrSinkConsumer struct {
```

Comment:
In the future if this gets copied consider to provide an util.

Reply:
Why create duplicates in receivers now? Maybe move to `internal` if you don't want to have this public.

Reply:
That is a good option, will do that. Indeed I am not sure about that code/api so prefer to have it non-public

Reply:
Done.